app/views/hikes/_hike.html.erb

```diff
@@ -2,5 +2,5 @@
   <td><%= link_to hike.name, hike_path(hike) %></td>
   <td><%= pluralize(hike.distance_in_km, 'Km') %></td>
   <td><%= pluralize(hike.time_in_hours, 'Hour') %></td>
-  <td><%= hike.difficulty_description %></td>
+  <td><%= t('difficulty'+"#{@hike.difficulty}") %></td>
```

Comment:
Personally I would write this as

`t(@hike.difficulty, scope: :difficulty)`

little bit cleaner, no interpolation.

Then your translations become:
```
en:
  difficulty:
    1: ...
    2: ...
    3: ...
```